doc/extensions/acrbuildext/azext_acrbuildext/__init__.py

```diff
@@ -42,4 +55,19 @@ def load_arguments(self, _):
             c.argument('no_logs', options_list=['--no-logs'], action='store_true', help='Do not show logs after successfully queuing the build.') 
         return self.argument_context
 
+        with self.argument_context('acr build-task') as c:
+            c.argument('registry_name', options_list=['--registry', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+            c.argument('build_task_name', options_list=['--name', '-n'], help='The name of the build task', completer=get_resource_name_completion_list(BUILD_TASK_RESOURCE_TYPE))
+            c.argument('image_name', options_list=['--image', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
+            c.argument('context', options_list=['--context'], help="The URL to a git repository.")
```

Comment:
From my understanding this `context` here is different from the `context` in `acr build`. Do we have to use the same parameter with different meanings?

Reply:
@northtyphoon

Reply:
context shouldn't just be "the url to a git repo" either, it's either the path to a Dockerfile -or- an url to a git repo

Reply:
context should be the same everywhere, it's either a path to a directory containing a docker file or a git url, have we deviated here?

Reply:
I think here it can only be a git repository. Yes if it can also support a docker file there is no issue here.

Reply:
this is quite different than docker actually, why aren't we using context configuration for Git URLs here, we should be able to use `abcdef.git#branchName:Context` - eliminates 2 parameters (source branch, docker file path) and it still makes sense as the context